rules/expensify.js

```diff
@@ -10,5 +10,12 @@ module.exports = {
         'rulesdir/prefer-underscore-method': 'error',
         'rulesdir/no-useless-compose': 'error',
         'rulesdir/prefer-import-module-contents': 'error',
+        'no-restricted-imports': ['error', {
+            'paths': [{
+                'name': 'react-native',
+                'importNames': ['Button', 'TouchableOpacity', 'Text'],
```

Comment:
`TouchableOpacity` could probably go. It's more of a wrapper that can be used to make stuff tappable and does not add styling (besides reducing element opacity on press). I think we have some usages that cannot easily be changed to `ExpensifyButton`.

Reply:
Ok sounds good. I'll just do `Text` and `Button`.

Reply:
Ok, updated.